rhizome/lib/vm_setup.rb

```diff
@@ -343,6 +337,59 @@ def cloudinit(unix_user, public_key, nics)
 }
 KEA_DHCP4_CONF
 
+    vp.write_kea_dhcp6_conf(<<KEA_DHCP6_CONF)
+{
+  "Dhcp6": {
+    "valid-lifetime": 4000,
+    "renew-timer": 1000,
+    "rebind-timer": 2000,
+    "data-directory": "/vm/#{@vm_name}/kea",
+    "lease-database": {
+      "type": "memfile",
+      "persist": true,
+      "name": "/vm/#{@vm_name}/kea/dhcp6.leases"
+    },
+    "loggers": [ {
+        "name": "kea-dhcp6",
+        "output_options": [ {
+            "output": "/vm/#{@vm_name}/kea/kea-dhcp6.log"
+        } ],
+        "severity": "DEBUG",
+        "debuglevel": 99
+    } ],
+    "subnet6": [
+      {
+        "interface": "#{nics.first[2]}",
+        "subnet": "#{guest_network}",
+        "pools": [
+          {
+              "pool": "#{guest_network.nth(2)}-#{guest_network.nth(1000)}"
+          }
+        ]
+        # "pd-pools": [
+        #   {
+        #     "prefix": "#{guest_network.network}",
+        #     "prefix-len": 80,
+        #     "delegated-len": 80
+        #   }
+        # ]
+      }
+    ],
+    "interfaces-config": {
+      "interfaces": ["*"],
```

Comment:
Should this be stapled down to a particular interface(s)? Can't problematic DHCP requests come from `vethi`? Also, any thoughts about how multiple DHCP configurations for TAP work?